### ISSUE TITLE
.shape warn in .oneOfType

### ISSUE THREAD

**Issue by RyoNkmr** [dwightjack/vue-types#30]
Hi,

```js
      const customType = VueTypes.oneOfType([
        VueTypes.shape({
          id: Number,
          name: VueTypes.string.isRequired,
        }),
        VueTypes.shape({
          id: Number,
          age: VueTypes.integer.isRequired,
        }),
        VueTypes.shape({}),
      ]);
      const validator = forceNoContext(customType.validator)
      console.log('valid: -------------------------------')
      expect(validator({ id: 1, name: 'John' })).toBe(true)
      expect(validator({ id: 2, age: 30 })).toBe(true)
      expect(validator({})).toBe(true)
      console.log('invalid: -------------------------------')
      expect(validator({ id: 2 })).toBe(false)
      console.log('-------------------------------')
```

this works good, but

```
LOG: 'vaild: -------------------------------'
WARN: '[VueTypes warn]: shape - at least one of required properties "name" is not present'
WARN: '[VueTypes warn]: shape - at least one of required properties "name" is not present'
WARN: '[VueTypes warn]: shape - at least one of required properties "age" is not present'
LOG: 'invalid: -------------------------------'
WARN: '[VueTypes warn]: shape - at least one of required properties "name" is not present'
WARN: '[VueTypes warn]: shape - at least one of required properties "age" is not present'
WARN: '[VueTypes warn]: shape - object is missing "id" property'
WARN: '[VueTypes warn]: oneOfType - value type should be one of "Object", "Object", "Object"'
WARN: '[VueTypes warn]: oneOfType - custom validation failed'
LOG: '-------------------------------'
```

invalid types raise warnings while`oneOfType` evaluates each `shape` until an valid type found.

**Comment by dwightjack** [comment]
Hi and thanks for filing an issue.

I think the problem here is that, while simple validation is _muted_ when checking a type with `oneOfType`, that's not the case for `shape`.

The offending line is [this]
(https://github.com/dwightjack/vue-types/blob/master/src/index.js#L195). I'll think about a way to fix it in a consistent way. If you have any suggestion feel free to comment or submit a PR.

**Comment by dwightjack** [comment]
Hi!

I just released v1.2.1 that should fix the problem.

Let me know if it works for you.

**Comment by RyoNkmr** [comment]
Hi! it works great now, thank you!
